src-tauri/src/output_loop.rs

```diff
@@ -17,6 +19,21 @@ const SERIAL_FPS: u32 = 30;
 const SACN_FPS: u32 = 100;
 const WLED_FPS: u32 = 30;
 
+// Event payloads for rendering results
+#[derive(Clone, Serialize)]
+struct DmxRenderEvent {
+    output_id: String,
+    frame: u32,
```

Comment:
We do not need to track the frame on the render events. Please remove references to the frame on the render output here and elsewhere.